src/platform/packages/shared/kbn-unified-metrics-grid/src/components/chart_container/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React, { forwardRef } from 'react';
+import type { ChartSize } from '../chart';
+import { ChartSizes } from '../chart';
+
+export const ChartContainer = forwardRef<
```

Comment:
Q: Why do we need the `ChartContainer` to be a separate component when it is used only in the `LensWrapper` context, or do we plan to extend it for the loading states?

Reply:
>Why do we need the ChartContainer to be a separate component when it is used only in the LensWrapper context
It's not, I use it all all my three charts too. 

https://github.com/elastic/kibana/pull/236635/files#diff-450217b0ab38b4ad608225acc6f8d048ef5d3775c32dd14fcd8af2214d711c40R91
https://github.com/elastic/kibana/pull/236635/files#diff-b59bb1de73e9a5c0a1e0161b2032c6a2628399784948acbf5ca6a1d1570dac0eR90
https://github.com/elastic/kibana/pull/236635/files#diff-69d857b69726421c23c5d351e88892df4a482763b78a0639990b5826487f6177R78